### PR TITLE
Add hera 1.0.

### PR DESCRIPTION
**Pull request by cbrueffer**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/hera/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  summary: hera is a bioinformatics tool that helps analyze RNA-seq data, providing base-to-base alignment BAM files, transcript abundance estimation, and fusion gene detection.
+
+package:
+  name: hera
+  version: 1.0
+
+requirements:
+  run:
+    - libgcc # [linux]
+    - python >=3
```
Comment (bgruening) [inline recipes/hera/meta.yaml]: Python2 should be skipped in the build section. 
```
build:
   skip: True  # [py2k]
```

File: recipes/hera/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+test:
+  commands: (hera | grep VERSION 2>&1) > /dev/null
+
+source:
+  fn: hera-v1.0.linux.tar.gz #[linux]
+  url: https://github.com/bioturing/hera/releases/download/v1.0/hera-v1.0.linux.tar.gz #[linux]
```
Comment (bgruening) [inline recipes/hera/meta.yaml]: Can we compile it from source instead of using precompiled binaries?

Comment (cbrueffer) [inline recipes/hera/meta.yaml]: Yes, that's what I started with initially before resorting to the binaries. The build script downloads and statically compiles a few libraries (e.g. htslib, libhdf5); needs some patching to use the bioconda versions. I'll have a look at it tomorrow.

File: recipes/hera/meta.yaml (hunk 1 above)
Comment (cbrueffer) [inline recipes/hera/meta.yaml]: Thanks!

**Comment by cbrueffer** [comment]
Superceded by #5380.

**Pull request closed without merging**